src/scripts/settings/config.js

```diff
@@ -0,0 +1,159 @@
+(function () {
```

Comment:
the intent of this is if there is necessary configuration in the future based on features, that this would be where the configuration would go.  but since it is just coach specific right now, should it still be changes to `cc-config.js`?